packages/core/config/babel-preset.js

```diff
@@ -55,6 +57,13 @@ module.exports = () => {
                 // List of supported proposals: https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#ecmascript-proposals
                 proposals: true,
               },
+              exclude: [
+                // Although preset-env includes class-properties
+                // but webpack 4 doesn't support the syntax when target supports and babel transpilation is skipped
+                // https://github.com/webpack/webpack/issues/9708
```

Comment:
I don't understand this comment. I think the English is too broken 😬 
Can we simplify it to this?
```suggestion
                // Although preset-env includes class-properties webpack 4 doesn't support it.
                // https://github.com/webpack/webpack/issues/9708
```

Reply:
Sure. I copied it :P